src/Sarif.Driver/OrderedFileSpecifier.cs

```diff
@@ -103,10 +103,18 @@ private IEnumerable<IEnumeratedArtifact> EnumeratedArtifacts()
             }
             else
             {
-                WriteFilesInDirectoryToChannel(directory, filesToProcessChannel, filter, new SortedSet<string>(StringComparer.Ordinal));
-
-                filesToProcessChannel.Writer.Complete();
-                directoryEnumerationTask = Task.CompletedTask;
+                directoryEnumerationTask = Task.Run(() =>
```

Comment:
>directoryEnumerationTask [](http://example.com/codeflow?start=16&length=24)

where is the test for this?

Reply:
where is the test for this --- working on it next. Want to give an early update that the issue has been found, or else will need Scott's view since he made this change.

Reply:
added